### ISSUE TITLE
Add web OAuth2 video to docs

### ISSUE THREAD

**Issue by MarkEdmondson1234** [MarkEdmondson1234/googleAuthR#205]
https://youtu.be/pBVAyU4pZOU

**Comment by noderboarder** [comment]
I followed this but it keep making errors like below. Any suggestions please? 

Warning: Error in : API returned: Legacy People API has not been used in project 'my-project-number' before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/legacypeople.googleapis.com/overview?project=my project number then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.

**Comment by MarkEdmondson1234** [comment]
You have to click on the link in the error message and enable the API for your project.

**Comment by noderboarder** [comment]
@MarkEdmondson1234 yes. I followed one, but I was not able to address the issues. I think there might be some problems in the APIs.

**Comment by MarkEdmondson1234** [comment]
I think this may be the now defunct Google+ API.  Depending on your use case you may want to look at the Google SignIn module instead https://code.markedmondson.me/googleAuthR/articles/google-authentication-types.html#googlesignin-module-example
